hoi/metrics/oinfo_zerolag.py

```diff
@@ -0,0 +1,123 @@
+from math import comb
+import jax
+import jax.numpy as jnp
+import numpy as np
+from scipy.special import ndtri
+import logging
+
+logger = logging.getLogger("frites")
+from hoi.oinfo.conn_oinfo_jax import combinations
+from hoi.oinfo.conn_oinfo_jax import oinfo_mmult
+
+
+def ctransform(x):  # frites.core
+    xr = np.argsort(np.argsort(x)).astype(float)
+    xr += 1.0
+    xr /= float(xr.shape[-1] + 1)
+    return xr
+
+
+def copnorm_1d(x):  # frites.core
+    assert isinstance(x, np.ndarray) and (x.ndim == 1)
+    return ndtri(ctransform(x))
+
+
+def copnorm_nd(x, axis=-1):  # frites.core
+    assert isinstance(x, np.ndarray) and (x.ndim >= 1)
+    return np.apply_along_axis(copnorm_1d, axis, x)
+
+
+def conn_oinfo_jax(
+    data,
+    y=None,
+    variables=None,
+    features=None,
+    minsize=3,
+    maxsize=5,
+    sort=True,
+    verbose=None,
+):
+    """Dynamic, possibly task-related oinfo.
+
+    Parameters
+    ----------
+    data : array_like
+            Standard NumPy arrays of shape (n_samples, n_features, n_variables)
+    y : array_like
+        The feature of shape (n_trials,) for estimating task-related O-info.
+    features : array_like | None
+        Array of region of interest name of shape (n_features,)
+    variables : array_like | None
+        Array of time points of shape (n_variables,)
+    minsize, maxsize : int | 3, 5
+        Minimum and maximum size of the multiplets
+
+    Returns
+    -------
+    oinfo : array_like
+        The O-info array of shape (n_multiplets, n_variables) where positive values
+        reflect redundant dominated interactions and negative values stand for
+        synergistic dominated interactions.
+    """
+    # ________________________________ INPUTS _________________________________
+    # inputs conversion
+    is_task_related = isinstance(y, (str, list, np.ndarray, tuple))
+    # kw_links = {"directed": False, "net": False}
+
+    # extract variables
+    # x, attrs = data.data, cfg["attrs"]
+    y, features, variables = (
```

Comment:
This block is not essential since the input type is not an `xarray.DataArray` anymore. We can remove it.

Reply:
```
y = data[0]
features = data[1]
variables = data[2]
```
So, are you suggesting to do this instead?

Reply:
The input data is going to be a standard NumPy array. You will not be able to access y, features etc. from the coordinates of a standard NumPy array.